### ISSUE TITLE
X20 RS Haptic stick feedback right is stronger than the left

### ISSUE THREAD

**Issue by frskyrc-ben** [FrSkyRC/ETHOS-Feedback-Community#4562]
For X20 RS Haptic stick Special function there is either a software or possible hardware bug.

The Right stick haptic feedback feels much stronger than the left and seems as if the right haptic feedback is activating ALL motors

To test, create two Special Functions - Haptic

Set the Left stick on one switch, and Right stick to another switch. Select the same Pattern and Strength for both, then toggle each on individually.

You will feel that the Left stick is much weaker and the Right stick is much more pronounced, possibly this is because it activates the case haptic in error.

I'm not sure if this is just a software bug that can be fixed in later version or if it might be a potential hardware bug.

**Comment by pmullen503** [comment]
My RS is like this.

**Comment by BJR981S** [comment]
Tested with 2 RSs and 2 AWs. The RSs behaved identically with right haptic significantly stronger than the left. The AWs behaved similarly only the right haptic was not as strong as the RS right Haptic.

I see little value in using the left haptic on maximum it would not be significant enough at the field even on 10 max setting. the min setting of 1 cannot be felt at all.

Test Procedure. Haptics allocated to switches. Settings 5 Dashes, repeat once, max strength of 10.

**Comment by bsongis-frsky** [comment]
I will reduce the RS right hapic with firmware.

**Comment by BJR981S** [comment]
Ah no thanks. Thats like saying I will give you lung cancer in your other lung to balance it up. Reducing it, so it's as bad as the left is not an answer. It needs the left increased, the left haptic is useless on all the RSs and AWs. I will simply no longer use individual haptics. For others with this issue if you want to turn down the right haptic it's easy just reduce the setting down it does not require a firmware update. Is to possible to increase the max on the left to 20 rather than its current max of 10? I only started to fiddle with the haptics as I am experimenting with Vario, Air Speed and GPS speed sensors. I was trying to set up a stick shaker stall warning.

**Comment by BJR981S** [comment]
Added note. The right haptic is not activating ALL motors. It does not activate the left haptic.

**Comment by frskyrc-ben** [comment]
> Added note. The right haptic is not activating ALL motors. It does not activate the left haptic.

Yes sorry I should have been more specific, what I meant was that it seemed like the Right Haptic was also activating the Case haptic (not ALL)

Also yes I agree we should not reduce the haptic feel of the right stick but instead, increase the feel of the left haptic.

I also agree that the difference feels less pronounced on AW but still noticeable, this may be due to the metal shell being less "resonant" than the plastic shell.

**Comment by bsongis-frsky** [comment]
From firmware the 2 haptic are exactly the same. AFAIK I can only reduce the maximum, if this is what you want ;)

**Comment by BJR981S** [comment]
> From firmware the 2 haptic are exactly the same. AFAIK I can only reduce the maximum, if this is what you want ;)

Not for me, Its easy enough to turn down the vibration from the Special Functions settings. If thats what you want.

Maybe a hardware mod to add another case haptic to match the right side. I would be up for that. After all the whole reason to have the haptic is to get your attention. If you cant feel it, its not much value.

**Comment by bsongis-frsky** [comment]
I cannot do anything, except make the second haptic less strong. I have stashed my changes in progress for now, you won't be happy if I push them ;)

**Comment by NorbertHeli** [comment]
No solution since 18 month? Mine is just new??

**Comment by pmullen503** [comment]
> No solution since 18 month? Mine is just new??

The haptic situation on the RS is a physical design problem. There is no software solution.

**Comment by NorbertHeli** [comment]
My X20RS is new and this design problem isn´t solved from FrSky since 18 month - thats what I want to say :-)

**Comment by NorbertHeli** [comment]
But a different Question - is there a issue, because I can porgramm LUA to adress haptic to left or right stick?
